MetaMask/metamask-extension
# fix: reset to native asset on account change

## **Description**

Fixes an issue when you changed accounts while sending an NFT.  It would keep the NFT asset, even though the new account doesn't own it.  Reset to the native asset instead.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24398?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/24391

## **Manual testing steps**

1. Prepare a transaction to send an NFT
2. Before clicking Next, change accounts
3. It should reset to ETH/native token. 

## **Screenshots/Recordings**



### **Before**


https://github.com/MetaMask/metamask-extension/assets/3500406/7628c507-a694-4bf9-8d1b-e828f7a8cdc7



### **After**


https://github.com/MetaMask/metamask-extension/assets/3500406/1ac60586-cf97-47bc-9f4e-1e518b8a053b



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
